compose/cli/main.py

```diff
@@ -232,13 +237,13 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
-            --allow-insecure-ssl    Allow insecure connections to the docker
-                                    registry
+            --allow-insecure-ssl    Deprecated - no effect.
```

Comment:
Yeah, the drawback (as well as the benefit) of using `docopt` is that you can't have undocumented command line flags.